data-processing/entities/sentences/types.py

```diff
@@ -1,8 +1,21 @@
 from dataclasses import dataclass
+from typing import List
 
 from common.types import SerializableEntity
 
 
 @dataclass(frozen=True)
 class Sentence(SerializableEntity):
     text: str
+    extended_tex: str
```

Comment:
Would it make sense to just set the `context_tex` property to the more refined context extracted in the code, instead of defining another property? The only reason that `context_tex` currently has a fixed offset before and after the sentence is because it was easy to do at the time 😉

Reply:
This is hard to decide because sometimes `context_tex` has more accurate context than `extended_tex`, so I kept both of them for now. Once we have a better tex normalized in the later version, I will keep one of them for sure.